pkg/resizer/csi_resizer.go

```diff
@@ -74,6 +74,11 @@ func NewResizerFromClient(
 		return nil, resizeNotSupportErr
 	}
 
+	_, err = supportsControllerSingleNodeMultiWriter(csiClient, timeout)
+	if err != nil {
+		return nil, fmt.Errorf("failed to check if plugin supports SINGLE_NODE_MULTI_WRITER capability: %v", err)
+	}
```

Comment:
why is this an error?

Reply:
If the `ControllerGetCapabilities` RPC returns an error I figure there are larger problems worth surfacing; this would also affect the node resize capability check.